examples/sample-docs/rdf/input.rdf

```diff
@@ -0,0 +1,1365 @@
+<rdf:RDF

```

Comment:
If possible, can we use a smaller rdf file? This file would increase the cloning size of the repository if it is included in the history.

Reply:
I added a smaller file.